src/mscorlib/src/System/Buffer.cs

```diff
@@ -618,5 +455,13 @@ public static unsafe void MemoryCopy(void* source, void* destination, ulong dest
             Memmove((byte*)destination, (byte*)source, checked((uint)sourceBytesToCopy));
 #endif // BIT64
         }
+        
+#if BIT64        
+        [StructLayout(LayoutKind.Sequential, Size = 16)]
+        private struct Block16 { }
+
+        [StructLayout(LayoutKind.Sequential, Size = 64)]
+        private struct Block64 { } 
```

Comment:
Does it work well / will it work well on ARM64?

Reply:
@vkvenkat Great work optimizing this. I didn't have much time to look over the code, but one thing I noticed-- why are there `Block16` / `Block64` structs but no `Block32`?

Reply:
Lol I just realized you made that comment a year ago. Still asking if the code is still there though